guides/v2.2/cloud/reference/cloud-composer.md

```diff
@@ -45,50 +37,27 @@ The workflow is as follows:
 
-### vendor/magento/ece-patches {#ece-patches}
-
-{% include cloud/ece-patches.md %}
-
-### vendor/magento/ece-tools {#ece-tools}
-
-{% include cloud/ece-tools.md %}
+## vendor/magento/ece-tools {#ece-tools}
+The `ece-tools` package, also known as _ece-tools_, is compatible with Magento version 2.1.4 and later to provide a rich set of features you can use to manage your {{site.data.var.ee}} project.  It contains scripts and {{site.data.var.ece}} commands designed to help manage your code and automatically build and deploy your Cloud projects.
 
-### vendor/magento/product-enterprise-edition {#cloud-composer-prodee}
+## vendor/magento/product-enterprise-edition {#cloud-composer-prodee}
 This {% glossarytooltip 7490850a-0654-4ce1-83ff-d88c1d7d07fa %}metapackage{% endglossarytooltip %} requires Magento application components, including modules, frameworks, themes, and so on.
 
 ## Base packages and file marshalling
-Magento contains two base packages, `magento/magento2-base` and `magento/magento2-ee-base`. These packages contain interstitial files that cannot be classified as extensions, themes, frameworks, or language packages; for example, sample server configuration files, {% glossarytooltip bf703ab1-ca4b-48f9-b2b7-16a81fd46e02 %}PHP{% endglossarytooltip %} entry points, and so on.
+Magento contains two base packages: `magento/magento2-base` and `magento/magento2-ee-base`. These packages contain interstitial files that cannot be classified as extensions, themes, frameworks, or language packages; for example, sample server configuration files, {% glossarytooltip bf703ab1-ca4b-48f9-b2b7-16a81fd46e02 %}PHP{% endglossarytooltip %} entry points, and so on.
 
 These files are location-dependent, and cannot reside in the `vendor` directory. They are distributed as part of the base packages, and they rely on hooks located in the `magento/magento-composer-installer` package, which marshals them to the appropriate locations.
 
-One way in which {{site.data.var.ece}} deploys differently than other Magento installations is that it does not marshal base packages on the Cloud environment. This could change in a future Cloud release, but for now, on the Cloud environment specifically, the marshalling functionality of `magento/magento-composer-installer` is disabled.
+One way in which {{site.data.var.ece}} deploys differently than other Magento installations is that it does not marshal base packages on the Cloud environment. This could change in a future Cloud release, but for now, on the Cloud environment—specifically, the marshalling functionality of `magento/magento-composer-installer`—it is disabled.
 
-Therefore, when upgrading to a new Cloud version or adding, removing, or changing any packages that rely on file marshaling, you must:
+When upgrading to a new Cloud version or adding, removing, or changing any packages that rely on file marshaling, you must:
```

Comment:
- Use the fully qualified product name via a Jekyll var
- Upgrading to a new "cloud" version? do you mean commerce version? ece-tools?